app/models/manageiq/providers/openstack/provider.rb

```diff
@@ -1,11 +1,11 @@
-class ProviderOpenstack < Provider
+class ManageIQ::Providers::Openstack::Provider < ::Provider
```

Comment:
Building on [@chessbyte's comment](https://github.com/ManageIQ/manageiq/pull/3664#discussion_r36220133) ... would it make more sense to have this just be `/manageiq/providers/openstack.rb`?  Then, we'd have: 

```ruby
  ManageIQ::Providers::Openstack.create(...)
```

instead of 

```ruby
  ManageIQ::Providers::Openstack::Provider.create(...)
```

Thoughts?